src/actinia_core/resources/ephemeral_processing_with_export.py

```diff
@@ -199,7 +199,7 @@ def __init__(self, rdc):
         self.storage_interface = rdc.create_storage_interface()
 
     def _export_raster(self, raster_name,
-                       format="GTiff",
+                       format="COG",
```

Comment:
If i saw it right, there are 2 more endpoints that export raster:
1. /locations/{location_name}/mapsets/{mapset_name}/raster_layers/{raster_name}/geotiff_async
2. /locations/{location_name}/mapsets/{mapset_name}/raster_layers/{raster_name}/geotiff_async_orig
They are also using `_export_raster` in the `AsyncEphemeralRasterLayerExporterResource` calss. So should they really use COG automatically? When they should use it, the docu should be changed:
- https://github.com/mundialis/actinia_core/blob/master/src/actinia_core/resources/raster_export.py#L56
- https://github.com/mundialis/actinia_core/blob/master/src/actinia_core/resources/raster_export.py#L131

Reply:
Both call the EphemeralRasterLayerExporter class, where 'GeoTIFF' was hardcoded as format. The gdal-driver check is now included here such that COG is chosen if the COG driver is available. Docu is adapted accordingly.